2021-09-22 00:11:39,525 INFO    MainThread:12395 [wandb_setup.py:_flush():69] setting env: {}
2021-09-22 00:11:39,525 INFO    MainThread:12395 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-09-22 00:11:39,525 INFO    MainThread:12395 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-09-22 00:11:39,525 INFO    MainThread:12395 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-22 00:11:39,525 INFO    MainThread:12395 [wandb_init.py:_log_setup():348] Logging user logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20210922_001139-l2zjae4u/logs/debug.log
2021-09-22 00:11:39,525 INFO    MainThread:12395 [wandb_init.py:_log_setup():349] Logging internal logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20210922_001139-l2zjae4u/logs/debug-internal.log
2021-09-22 00:11:39,525 INFO    MainThread:12395 [wandb_init.py:init():381] calling init triggers
2021-09-22 00:11:39,525 INFO    MainThread:12395 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/nerus_lenta/news_ner_dataset.csv', 'checkpoint_path': '/root/konst/model_checkpoints/ner_employment', 'model_path': 'ner_base_epoch_0_acc_0.9701_vs_0.9706_loss_0.1609_vs_0.1350.pth', 'batch_size': 80, 'epochs': 20, 'start_epoch': 1, 'lam': 0, 'patience': 2, 'factor': 0.5, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 0}
2021-09-22 00:11:39,525 INFO    MainThread:12395 [wandb_init.py:init():430] starting backend
2021-09-22 00:11:39,525 INFO    MainThread:12395 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-22 00:11:39,531 INFO    MainThread:12395 [backend.py:ensure_launched():135] starting backend process...
2021-09-22 00:11:39,534 INFO    MainThread:12395 [backend.py:ensure_launched():140] started backend process with pid: 12425
2021-09-22 00:11:39,535 INFO    MainThread:12395 [wandb_init.py:init():435] backend started and connected
2021-09-22 00:11:39,537 INFO    MainThread:12395 [wandb_init.py:init():494] updated telemetry
2021-09-22 00:11:39,538 INFO    MainThread:12395 [wandb_init.py:init():517] communicating current version
2021-09-22 00:11:39,897 INFO    MainThread:12395 [wandb_init.py:init():522] got version response 
2021-09-22 00:11:39,897 INFO    MainThread:12395 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-22 00:11:40,136 INFO    MainThread:12395 [wandb_init.py:init():557] starting run threads in backend
2021-09-22 00:11:40,245 INFO    MainThread:12395 [wandb_run.py:_console_start():1605] atexit reg
2021-09-22 00:11:40,245 INFO    MainThread:12395 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-09-22 00:11:40,246 INFO    MainThread:12395 [wandb_run.py:_redirect():1484] Redirecting console.
2021-09-22 00:11:40,249 INFO    MainThread:12395 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-22 00:11:40,249 INFO    MainThread:12395 [wandb_init.py:init():582] run started, returning control to user process
2021-09-22 00:23:33,783 INFO    MainThread:12395 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 255
2021-09-22 00:23:33,783 INFO    MainThread:12395 [wandb_run.py:_restore():1547] restore
